convention-develocity-gradle-plugin/README.md

```diff
@@ -9,18 +9,23 @@ Note the inline comments in the build and source code for things to adjust speci
 #### Using a dynamic version
 
 For builds using Gradle 6.9 and later, it's recommended to apply the convention plugin using a [dynamic version](https://docs.gradle.org/current/userguide/dynamic_versions.html).
-This approach ensures that the latest version of the plugin is resolved, up to the specified maximum version.
-For example, the following resolves the latest version of the plugin up to, but excluding, version `2.0.0`:
+This approach ensures that the latest version of the plugin is automatically used, but only for the specified major version.
+For example, the following will automatically use the latest `1.x` version of the plugin up to, but excluding, version `2.0`:
```

Comment:
@etiennestuder should we also change the preconfigured versions as `1.0-SNAPSHOT`?

https://github.com/gradle/develocity-build-config-samples/blob/8a6b83a02c5b29fb1b0f87f16cd7233e02da6a6c/convention-develocity-gradle-plugin/plugins/gradle-2-through-4/build.gradle#L8

https://github.com/gradle/develocity-build-config-samples/blob/8a6b83a02c5b29fb1b0f87f16cd7233e02da6a6c/convention-develocity-gradle-plugin/plugins/gradle-5-or-newer/build.gradle#L8

https://github.com/gradle/develocity-build-config-samples/blob/8a6b83a02c5b29fb1b0f87f16cd7233e02da6a6c/convention-develocity-maven-extension/extension/pom.xml#L7

Reply:
I would think so, for consistency. For that matter, do they even need to be snapshots?

Reply:
I think they should be snapshots. It reinforces the idea that these are samples and not production ready out-of-the-box.

I will adjust the versioning.

Reply:
I don't think they should be snapshots. It looks like we suggest people should refer to snapshots in their projects.

Reply:
Dropped `-SNAPSHOT` from the versioning in 88489b3134110c80a7d0be3d412742bebe42ff2f.